constants/constants.go

```diff
@@ -8,6 +8,11 @@ const CustomDateFormat = "2006-01-02"
 // ActionItemType is datatype for ActionItems's
 type ActionItemType string
 
+// constats to be used for task providers
+const (
+	PivotalTracker = "pivotal"
```

Comment:
We have this constant in the Pivotal tracker task provider.

Reply:
removed